### DISCUSSION TITLE
Device Audio and Mic Audio at the same time while screen capture in Android SDK

### DISCUSSION THREAD

**Discussion by USAMAWIZARD** [ant-media/Ant-Media-Server#5276] [General]
Could you please let me know if it is possible to share system audio and mic audio at the same time while streaming using screen capture for Android SDK ?

**Comment by mekya** [comment]
Thank you for the question @USAMAWIZARD.

I think the fastest way for doing this is sending system mic in another audio stream. So there will be two streams. One stream includes screen share + system audio and other stream will be mic audio. You can play the mic audio in the player as subtrack. 

If this solution works for you, we can share some code snippet and step by step instructions for you.

Cheers

**Comment by USAMAWIZARD** [comment]
Thanks for you answer @mekya Yes, That solution works for me.It would be very help full if you can link some code examples.

**Reply by mekya** [reply]
Let me create an issue from this discussion to provide some code samples.

**Reply by ghost** [reply]
Thank you @mekya for the input. Kindly share some code examples and step by step guide if possible.

**Reply by gargVader** [reply]
@mekya Can you please point me to the code samples? Thank you 🙏

**Reply by USAMAWIZARD** [reply]
@gargVader , unfortunatly we could not find some time to make a sample , if you want to increase the priority of the task please reach out to us at contact@antmedia.io
https://github.com/ant-media/Ant-Media-Server/issues/5285
